base/applications/mscutils/servman/precomp.h

```diff
@@ -43,6 +43,11 @@
 #define ORD_ASCENDING   1
 #define ORD_DESCENDING  -1
 
+#define SC_MANAGER_SUCCESS    0
+//https://docs.microsoft.com/fr-fr/windows/win32/debug/system-error-codes--1000-1299-
+#define ERROR_SERVICE_REQUEST_TIMEOUT          1053
+#define ERROR_EXCEPTION_IN_SERVICE             1064 
```

Comment:
Both constants are available here, you can just include header file. https://github.com/reactos/reactos/blob/77b8505393069f6a11a1524705d8838e06ace89a/sdk/include/psdk/winerror.h#L615

Reply:
Thank you for your help, I didn't know that
I'll implement.